joomla/joomla-cms
# Very mirror http -> https

On very mirror http -> https.

For sure the updater / Installer / CMS should not broken by this change.... else feel free to close. Thanks.
### How to test
- Install this package https://github.com/zero-24/joomla-cms/archive/https.zip
- Install with sample learn and sample testing SQL
- Make sure the installation breaks
- see the changed URLs e.g. issues.joomla.org uses now https as well as the move from http://community.joomla.org/events.html -> https://events.joomla.org
